libraries/aws_iam_policy.rb

```diff
@@ -85,6 +85,14 @@ def has_statement?(criteria = {})
     statement_match
   end
 
+  def statement_count
+    return false unless @policy_document
+    puts "DEBUG policy doc: #{@policy_document.inspect}"
+    document = JSON.parse(URI.decode_www_form_component(@policy_document.policy_version.document))
+    statements = document['Statement'].is_a?(Hash) ? [document['Statement']] : document['Statement']
```

Comment:
Is it possible to have the `Statement` key here a symbol?

Reply:
Hurm, I'll test it. Was following the pattern from existing 'has_statement' section just above.

Reply:
Updated JSON.parse options to create symbols.